packages/server/assets/automations/typedefs/automation.graphql

```diff
@@ -1,112 +1,118 @@
-  # resultVersions: [Version!]!
-
-  blobs: [String!]!
+  resultVersions: [Version!]!
   statusMessage: String
-  # its a {version: string; objectResults: Record<string, Array<{level: string; statusMessage: string}>>}
-  # where the record key is the object id
-  objectResults: JSONObject!
-}
-
-extend type Mutation {
-  automationMutations: AutomationMutations!
+  """
+  Current schema: {
+    version: "23.09.2023",
+    values: {
+      speckleObjects: Record<ObjectId, {level: string; statusMessage: string}[]>
+      blobIds?: string[]
+    }
+  }
+  """
+  results: JSONObject!
 }
 
-type AutomationMutations {
-  functionRunStatusReport(input: ModelAutomationRunStatusUpdateInput!): Boolean!
-    @hasServerRole(role: SERVER_USER)
-  # @hasStreamRole(role: STREAM_OWNER) # who can do this?
-  # @hasScope(scope: "automation:result") # TODO: add the scope constant
-  create(input: ModelAutomationCreateInput!): Boolean! @hasServerRole(role: SERVER_USER)
-  # @hasStreamRole(role: STREAM_OWNER) # TODO: check for stream ownership in the service
-  # @hasScope(scope: "automation:create") # TODO: add the scope constant
-}
-
-input ModelAutomationCreateInput {
+input AutomationCreateInput {
   projectId: String!
   modelId: String!
   automationName: String!
   automationId: String!
   automationRevisionId: String!
-}
-
-input ModelAutomationRunStatusUpdateInput {
-  versionId: String!
-  automationId: String!
-  automationRevisionId: String!
-  automationRunId: String!
-  # functionReleaseId: String!
-  functionRunStatuses: [FunctionRunStatusInput!]!
-}
-
-enum AutomationRunStatus {
-  INITIALIZING
-  RUNNING
-  SUCCEEDED
-  FAILED
-  # TIMEOUT needs to be handled as an error
+  webhookId: String
 }
 
 input FunctionRunStatusInput {
   # we cannot strictly require these values, cause local testers of function, wont have it...
   # Or should we?
   functionId: String!
   elapsed: Float!
-  runStatus: AutomationRunStatus!
+  status: AutomationRunStatus!
   contextView: String
   resultVersionIds: [String!]!
-  blobs: [String!]!
   statusMessage: String
-  # its a Record<string, {level: string; statusMessage: string}[]>
-  # where the record key is the object id
-  objectResults: JSONObject!
+  """
+  Current schema: {
+    version: "23.09.2023",
+    values: {
+      speckleObjects: Record<ObjectId, {level: string; statusMessage: string}[]>
+      blobIds?: string[]
+    }
+  }
+  """
+  results: JSONObject!
 }
 
-# extend type Subscription {
-#   automationRunStatusUpdated()
-# }
+input AutomationRunStatusUpdateInput {
+  versionId: String!
+  automationId: String!
+  automationRevisionId: String!
+  automationRunId: String!
+  functionRuns: [FunctionRunStatusInput!]!
+}
+
+type AutomationMutations {
+  functionRunStatusReport(input: AutomationRunStatusUpdateInput!): Boolean!
+    @hasServerRole(role: SERVER_USER)
+  # @hasStreamRole(role: STREAM_OWNER) # who can do this?
+  # @hasScope(scope: "automation:result") # TODO: add the scope constant
```

Comment:
@gjedlicska What about these scope/streamRole directives? Should I remove them or?

Reply:
the stream owner should go, but the scope is still something i'd want to discuss with Dim